CMakeLists.txt

```
@@ -86,22 +90,19 @@ Example on how to use :cmake:command:`rapids_make_logger`.
 
 .. code-block:: cmake
 
-  # Generate a logger for the namespace "rapids" and associate it with the
-  # export set "rapids-exports".
+  # Generate a logger for the namespace "rapids" with default level INFO and
```

Lets use a different example than the actual default.
```suggestion
  # Generate a logger for the namespace "rapids" with default level WARN and
```